R/save.R

```diff
@@ -1,6 +1,6 @@
 #' @rdname plot_carp
 #' @export
-saveviz <- function(x, ...) {
+saveviz <- function(x,frames_number,frame_rate, ...) {
```

Comment:
These need to be documented (for the methods) -- see how the other arguments are documented below.